### ISSUE TITLE
Maps: Map layout should show all events 

### ISSUE THREAD

**Issue by EmilyLuijbregts** [HTBox/allReady#2022]
This is a breakout from EPIC: MAPS: #2021 

As a user i want to see the map with all of the events that are categorised in one campaign or one calendar day. I should be able to choose between whether I want to see the campaign view or the calendar day in the map overview directly.

**Definition of Done:**
- This task is considered done when all users are able to see the campaign selected or the calendar day specified. 
- They should be able to change between the campaign and calendar options within the map feature
- The campaign view will show all of the registered pins in a chosen map location
- The date view should be in US Format (mm/dd/yyyy)

**Comment by stevejgordon** [comment]
@EmilyLuijbregts/@OhMcGoo Can I just check what "task" means here. It sounds like we mean "request" - i.e. the individual install locations requiring service? Tasks in allReady are a unit we use to define general work that will be done as part of an event and which volunteers can register for. i.e. a task might be "Fire alarm installer on 5th August 2017" for event "South Region Alarm Installs".

I'm not sure we'd need to map the tasks as they are used mainly for gaining volunteer signup. It's the requests that I assume we would want to map?

**Comment by OhMcGoo** [comment]
Yep @stevejgordon, that makes sense. The task would be to install smoke alarms on a certain day during which several requests would be fulfilled. Those requests need to be mapped, of course, so they can be grouped and assigned to a task.
